business_objects/record.py

```diff
@@ -309,19 +392,14 @@ def count_missing_rats_records(project_id: str, attribute_id: str) -> int:
     return result.c
 
 
-def count_missing_tokenized_records(project_id: str) -> int:
+def count_missing_tokenized_records(project_id: str) -> List[Any]:
     query = f"""
-    SELECT COUNT(*) c
-    FROM record r
-    LEFT JOIN record_tokenized rt
-        ON r.id = rt.record_id 
-        AND r.project_id = rt.project_id 
-        AND rt.project_id = '{project_id}'
-    WHERE r.project_id = '{project_id}' 
-    AND rt.id IS NULL    
+    SELECT COUNT(record_query.id)
```

Comment:
COUNT of a column should only be used for specific cases where we explicitly want to count that column.
This is due to a table scan for NULL values of the column since they wont be counted towards the result.

So since we want to count all entrys (and pkey can't be null) I'd go with COUNT(*).

Reply:
change to count(*)